packages/node-cli/src/services/ProjectInfoService.js

```diff
@@ -28,6 +28,7 @@ let CACHED_PROJECT_TYPE;
 module.exports = class ProjectInfoService {
 	constructor(projectFolder) {
 		assert(projectFolder);
+		CACHED_PROJECT_TYPE = null;
```

Comment:
same as before, I think that, at the end you want to have one cache per instance of the class, so it's better if you add it as a property in ProjectInfoService

Reply:
done